docs/guides/python.md

```diff
@@ -0,0 +1,163 @@
+# Python Developer's Guide
+
+When creating a Python function using the `func` CLI, the project directory
+looks like a typical Python project. Both HTTP and Event functions have the same
+template structure.
+
+```
+~/src/fn
+❯ kn func create -l python
+Project path: /home/developer/src/fn
+Function name: fn
+Runtime: python
+Trigger: http
+
+~/src/fn via 🐍 v3.8.5
+❯ tree
+.
```

Comment:
```suggestion
❯ tree
fn
```
This is what we are using in the other guides, but feel free to not use this suggestion :)